concent_api/utils/logging.py

```diff
@@ -1,11 +1,15 @@
-from logging                        import getLogger
+from enum import Enum
 
 from golem_messages.message import FileTransferToken
-from golem_messages.message.base    import Message
+from golem_messages.message.base import Message
```

Comment:
As I mentioned you in earlier pull requests. Don't correct style in imports which you are not changing

Reply:
done

Reply:
I think it's fine to correct it as long as it's next to the lines you're changing.

For example I much prefer if you leave this:

``` python
from golem_messages.message import FileTransferToken
from golem_messages.message.base import Message
from core.models import Subtask
from utils.helpers import get_field_from_message
```

rather than this:
``` python
from golem_messages.message import FileTransferToken
from golem_messages.message.base    import Message
from core.models import Subtask
from utils.helpers import get_field_from_message
```

the second one is the worst possible mix of both styles :)

We should avoid changing the style just because but if you're really changing something, there's no harm if you add a separate commit that cleans up the style around your change.